easy-ui-react/.stylelintrc

```diff
@@ -0,0 +1,13 @@
+{
+  "root": true,
+  "extends": ["@easypost/stylelint-easy-ui"],
+  "overrides": [
+    {
+      "files": ["src/**/*.scss"],
+      "customSyntax": "postcss-scss"
```

Comment:
when i tried to run the stylelint without this, i received this console message 

![Screen Shot 2023-03-01 at 3 17 37 PM](https://user-images.githubusercontent.com/44451175/222288770-0d663e31-12a3-4a6d-9058-92e2b6908fb8.png)

admittedly im not super familiar with it, but i went ahead and added `stylelint-config-recommended-scss` and created a separate override. although again, not totally sure if this is the ideal way

Reply:
ohh thanks for the heads up on that. forgot about modifying stylelint for scss. in that case, i think there are a couple things we need to do:

in `stylelint-easy-ui/package.json` uninstall (with `npm uninstall` so it saves to `package-lock.json`) the regular `recommended` package:

<img width="900" alt="image" src="https://user-images.githubusercontent.com/752942/222488127-596191e2-5b45-4e69-a867-5174ef63134f.png">

in `stylelint-easy-ui/index.js` replace the regular `recommended` package with the scss one:

<img width="834" alt="image" src="https://user-images.githubusercontent.com/752942/222488549-50cc3edd-3290-45d3-b17d-5b7c469105d6.png">

and with that package in place, it'll complain about including the `.scss` extension inside of a partial, so we need to remove that in `easy-ui-react/src/styles/_common.scss`:

<img width="884" alt="image" src="https://user-images.githubusercontent.com/752942/222488914-d70ed97f-5d57-4abd-b7b8-4f0c676ef7d4.png">

then with this in place for the whole project, we can remove `easy-ui-react/.stylelintrc`